gym_gridverse/gym_wrappers.py

```diff
@@ -0,0 +1,21 @@
+import gym
+
+class GymStateObservationWrapper(gym.Wrapper):
```

Comment:
I think this should be called GymStateWrapper;  otherwise StateObservation is not very clear from the name alone.  There's also other functionality I'd like this to have, but that can be added as a future commit.